.gitignore

```diff
@@ -3,6 +3,9 @@ node_modules
 /build
 /coverage
 packages/*/dist
+packages/**/*.js
+packages/**/*.d.ts
```

Comment:
Do we really need this line? Won't it be covered by the `*/dist` rule above?

Reply:
.d.ts files also are built into the package directories

Reply:
Same comment as above: Can we build `.d.ts` files in `dist/` or some other subdirectory?